ddtrace/internal/telemetry/data.py

```diff
@@ -76,7 +76,7 @@ def _get_application(key):
         "env": env or "",
         "language_name": "python",
         "language_version": _format_version_info(sys.version_info),
-        "tracer_version": get_version(),
+        "tracer_version": ddtrace.__version__,
```

Comment:
why this change?

Reply:
To avoid circular imports caused by the exception hooks